### PR TITLE
fix flutter analyze --dartdocs

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
I spent most of this afternoon trying to test this without success.

cc @tvolkert

File: .analysis_options_repo [hunk 1]
```diff
@@ -116,7 +116,7 @@ linter:
     # - prefer_interpolation_to_compose_strings # not yet tested
     - prefer_is_empty
     - prefer_is_not_empty
-    # - public_member_api_docs # this is the only difference from .analysis_options_repo
+    - public_member_api_docs # this is the only difference from .analysis_options_repo
```
Comment (tvolkert) [inline .analysis_options_repo]: So is that comment true anymore?

**Review by tvolkert — APPROVED** [review]
LGTM

File: .analysis_options_repo (hunk 1 above)
Comment (Hixie) [inline .analysis_options_repo]: The comment was always true. We just accidentally made it _not_ a difference a while back.
See https://github.com/flutter/flutter/pull/9330

**Pull request merged**